ios/MaasCore/Sources/MaasTheme/Theme/Spacing.swift

```diff
@@ -29,4 +26,16 @@ public struct Spacing {
     public static var `default`: Spacing {
         .init(value: CGFloat(SpacingScale.Default().GlobalMargin))
     }
+
+    public static var listItemHorizontal: Spacing {
+        .init(value: CGFloat(SpacingScale().sm))
+    }
+
+    public static var listItemVertical: Spacing {
+        .init(value: CGFloat(SpacingScale.Default().GlobalMargin))
+    }
+
+    public static var listItemGutter: Spacing {
+        .init(value: CGFloat(SpacingScale.Default().GlobalMargin))
+    }
```

Comment:
Shouldn't those be defined in Kotlin 🤔 I know its just mapped values thaat are declared in Kotlin, but android will need / use these as well, so would be still nice to have them declared in Kotlin. 
Ke probably don't need to expose API to override directly these values, but for our own use, we could have them declared once 🤔

Reply:
Agree, sounds good to me. 👍